src/_handler.ts

```diff
@@ -0,0 +1,13 @@
+import express from 'express';
+
+const app = express();
+app.set('trust proxy', true);
+
+app.get('*', async (_req, res) => {
+  res.setHeader('Content-Type', 'application/json');
```

Comment:
I think
```suggestion
  res.json().end({ hello: 'world' });
```
should also work.

Reply:
Ack.
It's going to change in the next PR anyway so I'm not touching it right now.
This PR is just so that @michaeladda can start working on the auto-deployment of this function while I work on it's actual implementation.